test/Icon.test.tsx

```diff
@@ -103,10 +103,8 @@ test('should have a testId for resilient UI changes', async () => {
 });
 
 test(`should have default title when no title is passed`, async () => {
-  const [, exceptionList] = await Promise.all([
-    preloadIcons(icons),
-    getExceptionList()
-  ]);
+  await preloadIcons(icons);
```

Comment:
If all icons in BI v.2 do not have a default title I think the `getExceptionList()` flow can be entirely removed